siliconcompiler/remote/client.py

```diff
@@ -341,9 +427,11 @@ def success_action(resp):
 
 
 ###################################
-def fetch_results_request(chip):
+def fetch_results_request(chip, node=''):
```

Comment:
node=None (to be consistent with the rest of SC

Reply:
The `node` value is used to assemble file paths, so if it is set to `None`, the file paths will include a `None` string where they are expected to be empty.

Reply:
Per offline discussion, I'm making these a required argument.